webserver/script/views.py

```diff
@@ -23,6 +26,16 @@ def post(self, request, format=None):
         sca.uploadToPostgres(baseline_profiles, controlled_profiles)
         return Response("Smart Charging run succeeded")
 
```

Comment:
api endpoint below for the costbenefit analysis runner. Runs the preprocessing module to prepare the load forecast csv profile to run through the CBA tool. Then uploads the CBA tool results to postgres CBA input tables.